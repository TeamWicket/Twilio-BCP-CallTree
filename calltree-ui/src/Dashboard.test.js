import Dashboard from './Dashboard';
import React from 'react';
import { shallow } from 'enzyme';

describe("Dashboard", () => {
    it("should match snapshot of Dashboard Component", () => {
        expect(shallow(<Dashboard />)).toMatchSnapshot();
    })
})
