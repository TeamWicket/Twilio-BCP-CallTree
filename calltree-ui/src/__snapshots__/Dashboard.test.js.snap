// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match snapshot of Dashboard Component 1`] = `ShallowWrapper {}`;
